### DISCUSSION TITLE
Community flag

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-starter#983] [Ideas]
I would like to include a flag in features to flag them as `community`.  E.g Camunda related features or [Micronaut Agorapulse libraries](https://github.com/agorapulse/micronaut-aws-sdk).  Similar to the `preview` flag.

thoughts?

**Comment by ilopmar** [comment]
:+1: for me!

**Comment by sdelamo** [comment]
@micronaut-projects/core-developers  thoughts?

**Reply by jameskleeh** [reply]
OK with me

**Comment by melix** [comment]
Ok for me.
